test/functional/rpc_blockchain.py

```diff
@@ -280,6 +280,14 @@ def _test_getblockheader(self):
         assert isinstance(int(header['versionHex'], 16), int)
         assert isinstance(header['difficulty'], Decimal)
 
+        # Test with verbose=False, which should return the header as hex.
+        header_hex = node.getblockheader(blockhash = besthash, verbose = False)
```

Comment:
It is preferred to pass named arguments without any spaces between the `=`. See https://www.python.org/dev/peps/pep-0008/#other-recommendations (point 5) and the discussion https://stackoverflow.com/questions/8853063/pep-8-why-no-spaces-around-in-keyword-argument-or-a-default-parameter-value.

Reply:
Thanks!